apps/tx-builder/package.json

```diff
@@ -8,6 +8,7 @@
     "@material-ui/core": "^4.12.3",
     "axios": "^0.24.0",
     "lodash": "^4.17.21",
+    "react-hook-form": "^7.25.3",
```

Comment:
There are [developer tools](https://react-hook-form.com/dev-tools) that you could install as a developer dependancy (rendering them only when you're in the developer env).

Reply:
done! :D